components/Heading/Heading.css

```diff
@@ -18,26 +32,58 @@
   margin: 0 10px 5px 10px;
 }
 
-.secondary {
```

Comment:
You wont want to delete the class names for the theme! Also, can you tie `fill` for the icon class names to CSS variables?